src/actions/plugins.js

```diff
@@ -17,3 +17,15 @@ export function getMarketplacePlugins(filter) {
     });
 }
 
+export function installMarketplacePlugin(id, version) {
```

Comment:
We wouldn't need this action either. The webapp would reference the new Client4 function direclty.